styles/base.css

```diff
@@ -1,4 +1,22 @@
 h1 {
-  font-size: 30px;
```

Comment:
Try using ems or another scaleable size method that will be more responsive for different viewport sizes.